pom.xml

```diff
@@ -168,6 +138,12 @@
                 <artifactId>opentelemetry-semconv</artifactId>
                 <version>${opentelemetry-alpha.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-sdk-testing</artifactId>
+                <scope>test</scope>
+                <version>${opentelemetry-alpha.version}</version>
+            </dependency>
```

Comment:
do we need this sdk testing dep? It's also in test scope but we don't have actual tests in the project.

Reply:
No, not needed. Removed